acstools/acs2d.py

```diff
@@ -21,6 +21,8 @@
     --> import acstools
     --> epar acs2d
 
+For help usage use exe_args=['--help']
```

Comment:
For nicer formatting please encapsulate "exe_args=['--help']" in double-backticks to generate rendering to the effect of `exe_args=['--help']`. (Tip: Don't use single-backtick because that will trigger `intersphinx` API linking, which will fail, and rendered differently.)